third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import java.util.List;
+
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+
+  private static final List<String> LOG_STRIP_SUFFIX = ImmutableList.of(
+      "dev.fine",
+      "dev.assert");
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    options.setStripTypes(Sets.newHashSet(LOG_STRIP_SUFFIX));
```

Comment:
yeah, i might just invoke `addCustomPass` which i think allows me to walk the AST. will try it out